process/block/metablock.go

```diff
@@ -881,7 +883,6 @@ func (mp *metaProcessor) checkAndProcessShardMiniBlockHeader(
 }
 
 func (mp *metaProcessor) createShardInfo(
-	maxMiniBlockHdrsInBlock uint32,
 	round uint32,
```

Comment:
put back the parameter, it is not needed to call MaxItemsToAdd in a continuous way as the processing goes. It is only calculated once per round.